ports/stm32/boards/stm32f412_af.csv

```diff
@@ -0,0 +1,116 @@
+Port,,AF0,AF1,AF2,AF3,AF4,AF5,AF6,AF7,AF8,AF9,AF10,AF11,AF12,AF13,AF14,AF15,
```

Comment:
Is this AF file exactly the same as an existing one, or is it F413-specific?

Reply:
I can't remember exactly as it was months ago when I added this support but when I created it I remember there to be differences with any other file and manually working through the Reference Manual. I also just did a diff with some chips in the same family and there was differences, so believe it is needed. Thanks.